da-indexer/da-indexer-logic/src/eigenda/repository/blobs.rs

```diff
@@ -70,6 +97,11 @@ pub async fn upsert_many<C: ConnectionTrait>(
         .on_empty_do_nothing()
         .exec(db)
         .await?;
+
+    if let Some(s3_storage) = s3_storage {
+        s3_storage.insert_many(data_s3_objects).await?;
+    }
```

Comment:
Does it make sense to run db batch insert and s3 upload in parallel? Something like `join!` can be used

Reply:
Updated in 3c310e1a